composer.json

```diff
@@ -63,6 +63,11 @@
     "extra": {
         "composer-exit-on-patch-failure": true,
         "drupal-scaffold": {
+            "file-mapping": {
+                "[project-root]/.gitignore": false,
```

Comment:
The `.gitignore` provided by this template replaces the `.gitignore` provided by Drupal scaffold as an "example" since our files is an "extended example".